source_code/main_mcu/src/utils.c

```diff
@@ -543,3 +543,23 @@ __attribute__( ( always_inline ) ) inline uint32_t utils_get_SP(void)
 #endif
 }
 
+/*! \fn     utils_switch_endiannes(time_t little_endian)
+*   \brief  Switches endiannes of a given time_t value by swapping order of bytes. If input is in little endian, the output will be big endian, and vice versa
+*   \param  input value in any endiannes
+*   \return input in switched endiannes
+*/
+time_t utils_switch_endiannes(time_t input)
```

Comment:
Please don't reinvent the wheel..:-) See main_mcu/src/ASF/sam0/utils/compiler.h: Swap64(u64)

Reply:
nice

Reply:
Good to know. Was not aware of that.

Reply:
Different question: Does that work in the emulator as well? When I add compiler.h to the QtCreator project of the emulator, I get errors, that "<parts.h>" is not found. And that remains even in case I add that parts.h to the project as well. To me now it looks like, that would work for in Atmel Studio, but not for the emulator.